tests/anvill_passes/src/Utils.cpp

```diff
@@ -84,7 +84,10 @@ std::unique_ptr<llvm::LLVMContext> CreateContextWithOpaquePointers(void) {
   auto context = std::make_unique<llvm::LLVMContext>();
 #if LLVM_VERSION_NUMBER < LLVM_VERSION(15, 0)
   context->enableOpaquePointers();
+#else
+  context->setOpaquePointers(true);
```

Comment:
Isn't this default behaviour? If not, this needs to be happen in a bunch more places in Anvill + my Remill change needs fixes too.